src/darwin/Framework/CHIP/zap-generated/MTRClusterConstants.h

```diff
@@ -2848,10 +2848,6 @@ typedef NS_ENUM(uint32_t, MTRAttributeIDType) {
         "Please use MTRAttributeIDTypeClusterTimeSynchronizationAttributeTimeSourceID", ios(16.1, 16.4), macos(13.0, 13.3),
         watchos(9.1, 9.4), tvos(16.1, 16.4))
     = 0x00000002,
-    MTRClusterTimeSynchronizationAttributeTrustedTimeNodeIdID MTR_DEPRECATED(
```

Comment:
These bits need changes to the Darwin availability metadata to avoid breaking API changes.  I'm happy to make those, but that would be easier if this PR were actually merged to tip.

Reply:
I will rebase soon and also push more work on this. I've been out the past few days.